### PR TITLE
EFRS-1186: Added ability to change the max image size

### PR DESCRIPTION
**Pull request by VolodymyrBushko**
Added ability to change the max image size via .env file for java and nginx config

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -57,6 +59,10 @@ services:
     depends_on:
       - compreface-api
       - compreface-admin
+    environment:
+      - CLIENT_MAX_BODY_SIZE=${client_max_body_size}
+    volumes:
+      - "./ui/nginx/templates:/etc/nginx/templates/"
```
Comment (pospielov) [inline docker-compose.yml:65]: our users won't have this folder.
You need to update [dockerfile](https://github.com/exadel-inc/CompreFace/blob/master/ui/docker-prod/Dockerfile), so this template will be in docker image.

File: .env [hunk 2]
```diff
@@ -12,6 +12,8 @@ enable_email_server=false
 save_images_to_db=true
 compreface_api_java_options=-Xmx4g
 compreface_admin_java_options=-Xmx1g
+max_file_size=10MB
+client_max_body_size=10M
```
Comment (pospielov) [inline .env:16]: what is the difference between these two variables?

**Review by pospielov — CHANGES_REQUESTED** [review]

File: .env (hunk 2 above)
Comment (VolodymyrBushko) [reply .env:16]: max_file_size - sets the total request size for a multipart/form-data.
client_max_body_size - sets the maximum allowed size of the client request body.

Should I join them in one?

Comment (pospielov) [reply .env:16]: no, let's leave them separate.
I just don't understand, why then in [java/admin/src/main/resources/application.yml](https://github.com/exadel-inc/CompreFace/pull/778/files#diff-d0bf48421385c221ee3c77956f0ea941b80f5cefb5d91678cb8e20435274ff52) you assign the same variable:
```     
max-file-size: ${MAX_FILE_SIZE:10MB}
max-request-size: ${MAX_FILE_SIZE:10MB}```

Comment (VolodymyrBushko) [reply .env:16]: I thought it can be ok in our case. I will split it

**Pull request merged**